src/niweb/niweb/settings/dev.py

```diff
@@ -76,3 +76,4 @@
 ########## END TESTING
 
 GOOGLE_MAPS_API_KEY = environ.get('GOOGLE_MAPS_API_KEY', 'no-apikey')
+CORS_ORIGIN_ALLOW_ALL = True
```

Comment:
Need to read up on CORS again, not sure if this is a problem (afaic remember  it is the default behavior).

Reply:
I had a look when I saw this and it is not default but it should be ok in this case as it is in dev. I guess it makes it possible to use cross domain calls between localhost and docker containers.

Reply:
Had not seen it was in dev, so yeah this is fine.